packages/remix/src/client/Interstitial.tsx

```diff
@@ -60,6 +60,7 @@ const createInterstitialHTMLString = (frontendApi: string, libVersion: string, d
                 script.setAttribute('data-clerk-frontend-api', '${frontendApi}');
                 script.async = true;
                 script.src = '${getScriptUrl(frontendApi, libVersion)}';
+                script.crossOrigin = '';
```

Comment:
🙃 Since setting the attribute name to an empty value, like crossorigin or crossorigin="", is the same as anonymous let's write `anonymous` to ease the future reader.